core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -143,9 +140,32 @@ public void hadoopShouldLoadFaultTolerantFileSystemWhenConfigured() throws Excep
     }
   }
 
+  @Test
+  public void hadoopShouldLoadFileSystemWithSingleZkUri() throws Exception {
+    org.apache.hadoop.conf.Configuration conf = getConf();
+    URI uri = URI.create(Constants.HEADER + "zk@zkHost:2181/tmp/path.txt");
+    org.apache.hadoop.fs.FileSystem fs = org.apache.hadoop.fs.FileSystem.get(uri, conf);
+
+    assertTrue(alluxio.Configuration.getBoolean(PropertyKey.ZOOKEEPER_ENABLED));
+    assertEquals("zkHost:2181", alluxio.Configuration.get(PropertyKey.ZOOKEEPER_ADDRESS));
+    assertTrue(fs instanceof FileSystem);
+  }
+
+  @Test
+  public void hadoopShouldLoadFileSystemWithMultipleZkUri() throws Exception {
+    org.apache.hadoop.conf.Configuration conf = getConf();
+    URI uri = URI.create(Constants.HEADER + "zk@host1:port1,host2:port2,host3:port3/tmp/path.txt");
```

Comment:
also test `zk@host1:port1;host2:port2;host3:port3/tmp/path.txt`?

Reply:
besides, I will also test the cases:
1. set `alluxio.zookeeper.enabled` and `alluxio.zookeeper.address` in system properties (check how `SystemPropertyRule` is used in tests), but specify nothing in uri, still expect zk uri.
1. set `alluxio.zookeeper.enabled` to `false` in conf or system properties, but set `alluxio.zookeeper.address` in URI, expect zk uri finally (i.e., URI should overwrites the other conf sources).
1. set `alluxio.zookeeper.enabled` to `true` and `alluxio.zookeeper.address` to `host1` in conf or system properties, 
but set `alluxio.zookeeper.address=host2` n URI, expect zk uri and host2 wins
other cases you can come up with to cover? Essentially, this conversion is kinda tricky, we should cover as many  cases as possible in unit tests.

Reply:
Added all the tests that test the functionality I add.